R/ResampleDesc.R

```diff
@@ -141,6 +149,53 @@ makeResampleDescRepCV = function(reps = 10L, folds = 10L) {
   makeResampleDescInternal("repeated cross-validation", iters = folds * reps, folds = folds, reps = reps)
 }
 
+makeResampleDescFixedCV = function(horizon = 1L, initial.window = .5, size = 100L, skip = .01  ) {
+  horizon = asInteger(horizon, lower = 1L, upper = Inf)
+  assertNumeric(initial.window, lower = 1E-15, upper = 1)
+  size = asInteger(size, lower = 1L, upper = Inf)
+  assertNumeric(skip, lower = 0L, upper = Inf)
+  initial.window.desc = floor(initial.window * size)
```

Comment:
> Also: What happens when size - horizon is smaller than initial.window.desc? Is it an error? Should the user be warned about that?

I think what we want to catch is when the `size - initial.window.size < horizon` as this does not leave enough data to create a test set